lib/gooddata/models/domain.rb

```diff
@@ -243,17 +243,30 @@ def create_users(list, default_domain = nil, opts = { :client => GoodData.connec
           begin
             user_data = user.to_hash.tap { |uh| uh[:login].downcase }
             domain_user = domain_users_cache[user_data[:login]]
+            user_login = user_data[:login]
             if !domain_user
               added_user = domain.add_user(user_data, opts)
+              GoodData.logger.info("Added new user=#{user_login} to domain=#{default_domain_name}.") if GoodData.logging_on?
```

Comment:
Please remove these `if`s. They are not needed in order to disable logging. It's already disabled after calling `GoodData.logging_off`.